Source/WebInspectorUI/UserInterface/Views/TimelineOverview.js

```diff
@@ -552,6 +552,11 @@ WI.TimelineOverview = class TimelineOverview extends WI.View
 
     _handleWheelEvent(event)
     {
+        // Ignore handling wheel events originating over the timelines tree outline to allow the timelines overview to be scrolled on small viewports.
+        // The result of compareDocumentPosition() is a bitwise mask: https://developer.mozilla.org/en-US/docs/Web/API/Node/compareDocumentPosition#node.document_position_contains
+        if (event.target.compareDocumentPosition(this._timelinesTreeOutline.element) & Node.DOCUMENT_POSITION_CONTAINS)
```

Comment:
NIT: why not something simpler like `this._timelinesTreeOutline.element.isAncestor(event.target)`?

Reply:
I can't find an `isAncestor()` on `HTMLElement` or in `Source/WebInspectorUI/UserInterface/Base/Utilities.js` Were you thinking of [WI.DOMNode.isAncestor()](https://github.com/WebKit/WebKit/blob/993ecb7ef85e18f280dd7876d150ef5054ac32f2/Source/WebInspectorUI/UserInterface/Models/DOMNode.js#L1006-L1018)?

Reply:
oh oops i meant `this._timelinesTreeOutline.element.contains(event.target)`